docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -1,5 +1,6 @@
 package com.nirima.jenkins.plugins.docker;
 
+import com.github.dockerjava.api.model.*;
```

Comment:
its only review point, not perfomance.

For example - how you can know where `PortBinding` located, if more than one `*` imports and no any import of `com.github.dockerjava.api.model.PortBinding`? You should go first to same package, then go to first `*`, then to second... (Or you should clone whole project to see it in IDE)